src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.js

```diff
@@ -53,12 +54,49 @@ function getDateBasedFromType(data) {
     }
 }
 
+const useValidateCustomDate = (data) => {
+    const localize = useLocalize();
+    const [customDateError, setCustomDateError] = useState('');
+    const [customTimeError, setCustomTimeError] = useState('');
+
+    const validate = () => {
+        const inputData = moment(data, 'YYYY-MM-DD HH:mm:ss');
+        const currentDate = moment().startOf('day');
+        const currentDateTimePlus2 = moment().add(1, 'minutes');
+
+        // Date validation
+        if (inputData.isBefore(currentDate)) {
+            setCustomDateError(localize.translate('common.error.dateInvalid'));
+        } else {
+            setCustomDateError('');
+        }
+
+        // Time validation
+        if (inputData.isBefore(currentDateTimePlus2)) {
+            setCustomTimeError(localize.translate('common.error.timeInvalid'));
```

Comment:
More of a UX comment, but it might be better to indicate what's actually wrong with the time input in the error, e.g."Enter a time at least 1 minute in the future"?